src/pages/home/Home.js

```diff
@@ -0,0 +1,85 @@
+import React, {useEffect, useState} from "react";
+import "./Home.css";
+import Header from "../../components/header/Header";
+import logo from "../../assets/reddit_logo.png";
+import axios from "axios";
+import {Link} from "react-router-dom";
+import titleLength from "../../helperfuncties/titleLength";
+import numberDots from "../../helperfuncties/numberDots";
+import Footer from "../../components/Footer";
+
+function Home() {
+    const [redditData, setRedditData] = useState("");
+    const [, toggleError] = useState(false);
+
+
+    useEffect(() => {
+        async function fetchData() {
+            toggleError(false);
+            try {
+                const result = await axios.get(`https://www.reddit.com/hot.json?limit=15`);
+                console.log(result.data.data.children);
+                setRedditData(result.data.data.children);
+
+            } catch (e) {
+                console.error(e);
+                toggleError(true);
+            }
+
+        }
+
+        fetchData();
+    }, []);
+
+    return (
+        <>
+            <Header
+                title="reddit">
+                <img src={logo} alt="Reddit Logo" width="170"
+                     height="170"/>
+            </Header>
+
+            <div className="inner_container">
+
+                <h2>Hottest posts</h2>
+                <h5>on Reddit right now</h5>
+                <main>
+                    {redditData && redditData.map((dataReddit) => {
```

Comment:
Als je de default waarde hebt aangepast, dan kun je dit versimpelen
```suggestion
                    {redditData.map((dataReddit) => {
```